.pipelines/dev-tunnels-ssh-ci.yaml

```diff
@@ -0,0 +1,299 @@
+trigger:
+  branches:
+    include:
+      - refs/heads/main
+
+name: $(date:yyyyMMdd)$(rev:.r)
+
+resources:
+  repositories:
+    - repository: self
+      type: git
+      ref: refs/heads/main
+
+variables:
+  - name: 'TeamName'
+    value: 'DevTunnels'
```

Comment:
Please enable codeql by adding `Codeql.Enabled` variable with value `true`

Reply:
See https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/configuring-codeql3000-ado-pipelines#enabling-codeql-3000 for details

Reply:
Updated